doc/classes/SpriteFrames.xml

```diff
@@ -170,6 +170,7 @@
 	</methods>
 	<members>
 		<member name="frames" type="Array" setter="_set_frames" getter="_get_frames">
+			Compatibility property, always equals to an empty array.
```

Comment:
Seems like the setter does something, but it's something weird. It sets the frames of the `"default"` animation, if any:
```
void SpriteFrames::_set_frames(const Array &p_frames) {

	clear_all();
	Map<StringName, Anim>::Element *E = animations.find(SceneStringNames::get_singleton()->_default);
	ERR_FAIL_COND(!E);

	E->get().frames.resize(p_frames.size());
	for (int i = 0; i < E->get().frames.size(); i++)
		E->get().frames.write[i] = p_frames[i];
}
Array SpriteFrames::_get_frames() const {

	return Array();
}
```

Reply:
`"default"` is the default animation in an `AnimatedSprite` though, so it's probably functional in most cases.
I don't know why the getter is unimplemented though, might be an oversight?

Reply:
Well, this line suggest that this method is not necessary anymore:
https://github.com/godotengine/godot/blob/4d3acbcb3a18520909ed6e36c3f9b5e681c2eb74/scene/2d/animated_sprite.cpp#L315